app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+
+from app.errors import (NotVaccinatedError, OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError
+        today = datetime.date.today()
+        if visitor["vaccine"]["expiration_date"] < today:
```

Comment:
I think you can remove this variable and compare it directly with datetime.date.today()